sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -2192,3 +2192,12 @@ def test_issue_25847():
     #acsc
     assert acsc(sin(x)/x).as_leading_term(x) == pi/2
     raises(PoleError, lambda: acsc(exp(1/x)).as_leading_term(x))
+
+def test_issue_23843():
```

Comment:
Check all possible combinations between `I` and infinities (`oo`, `-oo`) here for both `atan` and `acot`